src/main/kotlin/com/hibob/academy/dao/models.kt

```diff
@@ -1,3 +1,19 @@
 package com.hibob.academy.dao
 
+import java.time.LocalDate
+
+//return their name, dateOfArrival and company Id
+data class PetData(val petName: String, val dateOfArrival: LocalDate, val companyId: Long)
+
+//return their name, employee Id and company Id.
+data class OwnerData(val ownerName: String, val employeeId: Long, val companyId: Long)
+
+//pets by a given type (represented by enum)
+enum class TYPE{
```

Comment:
This is bad name, it doesn't say anything regarding what it is representing.
Lets think of a better name to this enum.

Reply:
I Change my enum class TYPE name to a more unique and representing name- PetType